embedded-compositor/dbusinterface.h

```diff
@@ -59,6 +59,7 @@ struct TaskSwitcherEntry {
   QString label;
   QString icon;
   uint32_t pid;
+  int32_t sortIndex;
```

Comment:
why is the `sortIndex` sign ? In my opinion it make no sense to allow the negative values.

Reply:
ist erledigt (5e6e15fc65b3ca7f9af2dfd56c539048fc051c1f)